.travis.yml

```diff
@@ -1,5 +1,15 @@
+sudo: false
+
 language: node_js
+
 node_js:
-  - "stable"
-  - "5"
-  - "4"
+  - 6
+  - 4
+
+cache:
+  directories:
+    - node_modules
+
+install:
+  - npm i -g npm@latest
```

Comment:
The general pattern is that travis ci versions only support LTS versions of node, which are 6 and 4. 5 and lower are dropped in newer projects and are supported by large projects that have many users.